spec/controllers/proposals_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+describe ProposalsController, type: :controller do
+  describe 'Get new' do
+    let!(:pq) { FactoryBot.create(:pq) }
```

Comment:
Does this need to be `let!` or would `let` work so that it only creates the object when needed?

Reply:
Amended as per suggestion